components/archiver/zip/Makefile

```diff
@@ -36,32 +37,32 @@ COMPONENT_BUGDB=	utility/zip
 
 LD_OPTIONS=
 
-include $(WS_TOP)/make-rules/prep.mk
-include $(WS_TOP)/make-rules/justmake.mk
-include $(WS_TOP)/make-rules/ips.mk
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/justmake.mk
+include $(WS_MAKE_RULES)/ips.mk
 
 # remove the unwanted file.
 COMPONENT_PREP_ACTION = rm -f $(SOURCE_DIR)/match.S
 
 # The Makefile we want is under unix/Makefile
-COMPONENT_BUILD_ARGS += -f unix/Makefile
-COMPONENT_BUILD_TARGETS += generic CC=$(CC)
-COMPONENT_INSTALL_ARGS += -f unix/Makefile
+COMPONENT_BUILD_ARGS    += -f unix/Makefile
+COMPONENT_BUILD_TARGETS += generic CC="$(CC) -m64"
```

Comment:
Could use the CC_BITS macros

Reply:
Maybe this works only for autotools? With `CC_BITS = -m64` it build as 32-bit:
```
/usr/gcc/6/bin/gcc -c -I. -DUNIX -O3 -DBZIP2_SUPPORT -DUIDGID_NOT_16BIT -DLARGE_FILE_SUPPORT -DUNICODE_SUPPORT -DHAVE_DIRENT_H -DHAVE_TERMIOS_H zip.c
```

Reply:
I meant to replace -m64 with $(CC_BITS)